provenance-checker/base/openshift-templates/provenanceChecker-template.yaml

```diff
@@ -0,0 +1,143 @@
+          - name: THOTH_LOG_ADVISER
+            value: "${THOTH_LOG_ADVISER}"
+          - name: THOTH_FORCE_SYNC
+            value: "${THOTH_FORCE_SYNC}"
+          - name: ceph_bucket_prefix
+          - name: ceph_bucket_name
+          - name: ceph_host
+          - name: deployment_name
+      volumes:
+        - name: workdir
+          emptyDir: {}
+        - name: output
+          emptyDir: {}
+      templates:
+        - name: "provenance-checker"
+          archiveLocation:
+            archiveLogs: true
+          dag:
+            tasks:
+              - name: "provenance-check"
+                templateRef:
+                  name: "provenance-check"
+                  template: "provenance-check"
+                continueOn:
+                  failed: true
+                arguments:
+                  parameters:
+                    - name: THOTH_PROVENANCE_CHECKER_JOB_ID
+                      value: "{{workflow.parameters.THOTH_PROVENANCE_CHECKER_JOB_ID}}"
+                    - name: THOTH_DOCUMENT_ID
+                      value: "{{workflow.parameters.THOTH_DOCUMENT_ID}}"
+                    - name: THOTH_WHITELISTED_SOURCES
+                      value: "{{workflow.parameters.THOTH_WHITELISTED_SOURCES}}"
+                    - name: THOTH_ADVISER_REQUIREMENTS
+                      value: "{{workflow.parameters.THOTH_ADVISER_REQUIREMENTS}}"
+                    - name: THOTH_ADVISER_REQUIREMENTS_LOCKED
+                      value: "{{workflow.parameters.THOTH_ADVISER_REQUIREMENTS_LOCKED}}"
+                    - name: THOTH_ADVISER_METADATA
+                      value: "{{workflow.parameters.THOTH_ADVISER_METADATA}}"
+                    - name: THOTH_LOG_ADVISER
+                      value: "{{workflow.parameters.THOTH_LOG_ADVISER}}"
+                    - name: THOTH_S3_ENDPOINT_URL
+                      value: "{{workflow.parameters.ceph_host}}"
+                    - name: THOTH_CEPH_BUCKET_NAME
+                      value: "{{workflow.parameters.ceph_bucket_name}}"
+                    - name: THOTH_CEPH_BUCKET_PREFIX
+                      value: "{{workflow.parameters.ceph_bucket_prefix}}"
+                    - name: THOTH_DEPLOYMENT_NAME
+                      value: "{{workflow.parameters.deployment_name}}"
+              - name: "graph-sync-provenance-check"
+                dependencies:
+                  - "provenance-check"
+                templateRef:
+                  name: "graph-sync"
+                  template: "graph-sync"
+                arguments:
+                  artifacts:
+                    - name: outputdocument
+                      from: "{{tasks.provenance-check.outputs.artifacts.outputdocument}}"
+                  parameters:
+                    - name: "THOTH_DOCUMENT_ID"
+                      value: "{{workflow.parameters.THOTH_DOCUMENT_ID}}"
+                    - name: "THOTH_FORCE_SYNC"
+                      value: "{{workflow.parameters.THOTH_FORCE_SYNC}}"
+
```

Comment:
the extra newline is causing the pre-commit failure.

Reply:
Should be fixed, thanks for pointing this out! :+1: